clients/mobile/Makefile

```diff
@@ -0,0 +1,81 @@
+# Define environment variables
+ENV_FILE=.env
+APP_NAME=net.bloodconnect.app
+EAS_CONFIG=eas.json
+
+# Default build profile (can be overridden via CLI)
+BUILD_PROFILE ?= development
```

Comment:
are you sure this gap should be here? AFAIK BUILD_PROFILE?=development
is the correct way

Reply:
if BUILD_PROFILE not passing from the cli it will default set as development. I think it's correct. I tested it on my machine.

Reply:
i meant the spaces around it.